app/modules.go

```diff
@@ -220,37 +170,49 @@ func orderBeginBlockers() []string {
 	}
 }
 
+func OrderEndBlockers(allModuleNames []string) []string {
+	ord := partialord.NewPartialOrdering(allModuleNames)
+	// Epochs must run after all other end blocks
+	ord.LastElements(epochstypes.ModuleName)
+	// txfees auto-swap code should occur before any potential gamm end block code.
+	ord.Before(txfeestypes.ModuleName, gammtypes.ModuleName)
+	// only remaining modules that aren;t no-ops are: crisis & govtypes
+	// we don't care about the relative ordering between them.
+
+	return ord.TotalOrdering()
+}
+
 // orderEndBlockers Tell the app's module manager how to set the order of
 // EndBlockers, which are run at the end of every block.
 var orderEndBlockers = []string{
-	lockuptypes.ModuleName,
+	lockuptypes.ModuleName, // no-op
 	crisistypes.ModuleName,
 	govtypes.ModuleName,
```

Comment:
What about these 2?

Reply:
No, they do work. However, their ordering does not matter as far as I know